packages/block-library/src/calendar/edit.js

```diff
@@ -8,69 +8,42 @@ import memoize from 'memize';
  * WordPress dependencies
  */
 import { Disabled } from '@wordpress/components';
-import { Component } from '@wordpress/element';
-import { withSelect } from '@wordpress/data';
+import { useSelect } from '@wordpress/data';
 import ServerSideRender from '@wordpress/server-side-render';
 
-class CalendarEdit extends Component {
-	constructor() {
-		super( ...arguments );
-		this.getYearMonth = memoize( this.getYearMonth.bind( this ), {
-			maxSize: 1,
```

Comment:
Any particular reason for the removal of the `maxSize` argument passed to `memoize`?

Reply:
No. It does not seem to harm anything.

Reply:
it's a memory optimization, I think. Not sure how worth it.

Reply:
Let me know if you want it back.

Reply:
It looks like the `maxSize` argument has been there since the block was added. Pinging @jorgefilipecosta, since he authored the original block.

Reply:
Jorge is busy right now, so I looked into it myself. I don't think there's any reason for the argument to be there, so it's fine that it's been removed.